load/InputFileset.py

```diff
@@ -405,7 +405,10 @@ def batchName(self):
 			if len(self.filesetId) < 10:
 				print ("DEBUG: text filesetid less than 10 characters long: " + self.filesetId)
 
-			return self.textFilesetId()
+			if self.subTypeCode() in {'text_html', 'text_json', 'text_usx'} :
```

Comment:
batchName method has been fixed to avoid to overlook adding the usx fileset to the output list.

Reply:
@vichugofsl we didn't discuss this earlier... do you think this is needed, and if so, why?